### ISSUE TITLE
LOCATION and TBLPROPERTIES clauses in CREATE TABLE LIKE statement

### ISSUE THREAD

**Issue by lapter57** [xnuinside/simple-ddl-parser#125]
Hi! If i create a table using the LIKE statement and add LOCATION and/or TBLPROPERTIES clauses to it, then returns `[]`.
```python
stmt = 'CREATE EXTERNAL TABLE IF NOT EXISTS schema.specific_table LIKE schema.table_template LOCATION "/path/to/table" TBLPROPERTIES ("external.table.purge" = "true")'
DDLParser(stmt).run(output_mode="hql")
```

**Comment by xnuinside** [comment]
@lapter57 , hi! thanks for reporting, will take a look at the evening

**Comment by xnuinside** [comment]
@lapter57  hi again! sorry for the delay, fix released with version 0.26.1, test added - https://github.com/xnuinside/simple-ddl-parser/blob/main/tests/test_hql_output_mode.py#L2251 if will be anything new - feel free to open new issue :) and thanks for report one more time!
